src/locales/ko/modifier-type.ts

```diff
@@ -70,7 +70,7 @@ export const modifierType: ModifierTypeTranslationEntries = {
       description: "자신의 포켓몬의 HP를 모두 회복한다.",
     },
     "AllPokemonFullReviveModifierType": {
-      description: "자신의 포켓몬의 HP를 기절해 버렸더라도 모두 회복한다.",
+      description: "자신의 포켓몬 중, 기절해 버린 포켓몬 전원의 HP를 완전히 회복한다.",
```

Comment:
```suggestion
      description: "기절해 버린 포켓몬 전원의 HP를 완전히 회복한다.",
```
사실 이렇게 쓰는 게 더 명확할 것 같은데, 한 번 고려부탁드립니다 🤔

Reply:
저는 이 버전(기절해 버린 포켓몬 전원의 HP를 완전히 회복한다)이 더 자연스러운 것 같습니다 :)

Reply:
![image](https://github.com/pagefaultgames/pokerogue/assets/19601207/4101dede-8e8a-49bc-b842-5bf8f6b8a97f)
저도 아군 포켓몬임을 명시할 필요가 없다고 생각합니다 😃

Reply:
아침부터 ㄷㄷㄷ ㅋㅋㅋㅋㅋ 감사합니다 의견 반영했어요~ 체크하시고 확인부탁드립니다 @malloc3141 @EnochG1